### PR TITLE
Node.js app for scanning articles removed from their parent pages.

### PR DESCRIPTION
**Pull request by seanmtracey**
Also contains Tesseract binary for EC2 instances.

### CODE REVIEW

File: scanner/index.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
+require('node-babel')();
+
+const dotenv = require('dotenv').config();
```
Comment (Lily2point0) [inline scanner/index.js]: dotenv not used

File: scanner/bin/dynamo.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+function describeTable(table){
+	return new Promise( (resolve, reject) => {
+		if(table === undefined || table === null){
+			reject(`'table' argument is ${table}`);
+		} else {
+			Dynamo.describeTable({
+				TableName : table
+			}, (err, result) => {
+				if(err){
+					reject(err);
```
Comment (Lily2point0) [inline scanner/bin/dynamo.js:17]: Promise rejection handling?

File: scanner/index.js [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+			const file = fs.createWriteStream(destination);
+			debug(destination)
+			S3.getObject({
+				Bucket : 'ftlabs-archives-snippets',
+				Key : resource
+			}).createReadStream().pipe(file);
+	
+			file.on('error', function(err){
+				debug('Error writing file', err);
+				debug(err);
```
Comment (Lily2point0) [inline scanner/index.js]: logging `err` twice

File: scanner/index.js [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+									;
+								})
+							;
+						}
+
+					})
+				;
+			}
+		
+		}, 1000);
```
Comment (Lily2point0) [inline scanner/index.js]: Set the interval time as a var so it can be changed

File: scanner/bin/dynamo.js (hunk 2 above)
Comment (seanmtracey) [reply scanner/bin/dynamo.js:17]: The error is handled in the app that calls the module, not in the module itself. https://github.com/ftlabs/ftda/blob/0143e8440cc135a30c2e922274691ae2c77f8787/scanner/index.js#L172

**Pull request merged**